src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -77,11 +73,20 @@ const ChatDetailHeader = ({
   type,
   channel
 }) => {
-  const bgHeaderStyle = () => {
-    if (type === SIGNED) return styles.bgDarkBlue;
-    return styles.bgBondiBlue;
+  const contentColor = () => {
+    if (type === SIGNED) return colors.black;
+    return colors.white;
   };
 
+  const animatedStyle = useAnimatedStyle(() => {
+    return {
+      backgroundColor: withTiming(type === ANONYMOUS ? colors.anon_primary : colors.white, {
+        duration: 400,
+        easing: Easing.inOut(Easing.quad)
+      })
+    };
+  });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces an animated style for the header background color, replacing the previous static styles. The animation changes the background color over a duration of 400ms using an in-out quad easing function. This should provide a smoother transition when the chat type changes. However, it's important to test this change thoroughly to ensure it doesn't introduce performance issues, especially on lower-end devices.



<!-- This is an auto-generated comment by OSS CodeRabbit -->